cli/command_completions_doc_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli_test
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/abcxyz/pkg/cli"
+	"github.com/posener/complete/v2"
+	"github.com/posener/complete/v2/predict"
+)
+
+type SingCommand struct {
+	cli.BaseCommand
+
+	flagSong string
+	flagFade time.Duration
+	flagNow  int64
+}
+
+func (c *SingCommand) Desc() string {
+	return "Sings a song"
+}
+
+func (c *SingCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options] PATH
+
+  Sings the given song at the audio file.
+`
+}
+
+// PredictArgs is an optional interface which will predict argument values. If
+// omitted, no arguments are suggested.
+func (c *SingCommand) PredictArgs() complete.Predictor {
+	return predict.Files("*.mp3")
+}
+
+func (c *SingCommand) Flags() *cli.FlagSet {
+	set := cli.NewFlagSet()
+
+	f := set.NewSection("Song options")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "song",
+		Aliases: []string{"s"},
+		Example: "Itsy Bitsy Spider",
+		Target:  &c.flagSong,
+		Predict: predict.Set{"Happy Birthday", "Twinkly Twinkle Little Star"},
+		Usage:   "Numeric value by which to increment between each number.",
```

Comment:
This usage string seems inconsistent with the rest of the fields, which are all about songs, not numbers

Reply:
I was having trouble coming up with an example. Fixed the usage.